en/02-javascript/0202-modelo-de-datos/index.md

```diff
@@ -134,13 +134,13 @@ point['x'] = 0;
 point['y'] = 0;
 ```
 
-If you access a **non-extant property,** you will obtain the value `undefined`:
+If you access a **non-existent property**, you will obtain the value `undefined`:
 
 ```js
 var label = point.label; // will be undefined. Check it out with typeof.
 ```
 
-We can create new properties at any time by assigning a value to them.

```

Comment:
Punctuated as in the original Spanish text. I'm pretty sure your change is the correct way, but I can't be 100% sure that the code segment under it is demonstrative. Please advise, @delapuente ?